cel/options.go

```diff
@@ -399,6 +399,16 @@ func registerFiles(reg customTypeRegistry, files *protoregistry.Files) error {
 // ProgramOption is a functional interface for configuring evaluation bindings and behaviors.
 type ProgramOption func(p *prog) (*prog, error)
 
+// Adds one or more ProgramOptions to the environment.
+//
+// The options are applied before any other options when creating a program for evaluation.
+func ProgOpts(options ...ProgramOption) EnvOption {
```

Comment:
Maybe call this `DefaultProgramOptions`? Alternatively, you could pass an `Activation` into `FoldVariablesWithKnownValues` and remove the `ProgOpts` option?

Reply:
Done.